packages/forms/resources/views/components/builder.blade.php

```diff
@@ -155,6 +155,8 @@
                                     'flex divide-x rtl:divide-x-reverse',
                                     'dark:divide-gray-700' => config('forms.dark_mode'),
                                 ])>
+                                    <x-forms::up-and-down-buttons component="builder" :is-item-movement-disabled="$isItemMovementDisabled" :loop="$loop" :state-path="$getStatePath()" :uuid="$uuid" />
```

Comment:
Please don't extract this Blade component, the code can be copied between each view which makes it easier to override the behaviour in the view for just 1 component

Reply:
Done. Just to understand it better for the next time: What is the benefit of not making it a component? This sorting behaviour logic is an atomic behaviour which can be easily isolated and it is already used by two individual components (repeater and builder). I don't get the benefit of copy+pasting code, especially if it has 40+ lines?

Making it a custom component, increases (in my eyes) the readability and most important: it makes it much more easy to overwrite the component in the app. For example if someone would change the icon of it, you will have to extend not only two files, but the whole blade template of repeater/builder.

Where is my mistake?

Reply:
Its not a mistake, just my preference in this situation.

1) none of the other buttons are extracted yet, would prefer to do them all at once
2) in v3 we're extracting the buttons to actions anyway (probably), so this new view will just need to be removed again
3) as I said before, easier to override without having to look up a separate view